Point.cpp

```diff
@@ -2,9 +2,12 @@
 
 using namespace std;
 
-void Point::draw(char ch) const
+void Point::draw(char ch,bool colored) const
 {
-	int colorShape = getSerialNumber() % 6 + 1;
+	int colorShape = 7;
+	if (colored) {
+		colorShape = getSerialNumber() % 7 + 1;
```

Comment:
Changing it to 7 doesn't change the options to contain white ? do we want that ?

Reply:
yes, why not?